node/src/main/scala/coop/rchain/node/api/WebApi.scala

```diff
@@ -330,13 +351,14 @@ object WebApi {
     case UnforgDeployer(name) => GDeployerIdBody(GDeployerId(name.unsafeHexToByteString))
   }
 
+  private def unforgToPar(unforg: RhoUnforg): Par =
```

Comment:
This name should have suffix `Proto`.

Reply:
[Done](https://github.com/rchain/rchain/pull/3718/commits/d9f613486bb84a973be0fa79e11fd61a3a770066).